sections/intro.tex

```diff
@@ -1,6 +1,290 @@
+of this kind of analysis is the ability to glean and communicate simplified
+summaries of patient needs to stakeholders within a healthcare
+system~\cite{Vuik2016b, Yoon2020}. For instance, clinical profiling often
+forms part of the wider analysis where each segment can be summarised in a
+phrase or infographic~\cite{Vuik2016a, Yan2019}.
+
+The review for this work identified three commonplace groups of patient
+characteristics used to segment a patient population: their system
+utilisation metrics, their clinical attributes and their pathway. The latter
+is not used to segment the patients directly but rather groups their movements
+through a healthcare system. This is typically done via process
+mining.~\cite{Arnolds2018}~and~\cite{Delias2015} demonstrate how this technique
+can be used to improve the efficiency of a hospital system as opposed to
+tackling the more relevant issue of patient-centred care. The remaining
+characteristics can be segmented with a number of techniques but recent works
+tend to use unsupervised methods, typically latent class analysis (LCA) or
+clustering~\cite{Yan2018}.
+
+LCA is a statistical, model-based method used to identify groups (called latent
+classes) in data by relating its observations to some unobserved (latent),
+categorical attribute. This attribute has multiple categories, each
+corresponding to a latent class. The discovered relations are then used to
+separate the observations into latent classes according to their maximum
+likelihood class membership~\cite{Hagenaars2002,Lazarsfeld1968}. This method has
+proved useful in the study of comorbidity patterns as
+in~\cite{Larsen2017,Kuwornu2014} where combinations of demographic and clinical
+attributes are related to various subgroups of chronic diseases.
+
+Similarly to LCA, clustering identifies groups (clusters) in data to produce a
+labelling of its instances. However, clustering includes a wide variety of
+methods where the common theme is to maximise homogeneity within, and
+heterogeneity between, each cluster~\cite{Everitt2011}. The \(k\)-means paradigm
+is the most popular form of clustering in literature. The method iteratively
+partitions numerical data into \(k \in \mathbb{N}\) distinct parts where \(k\)
+is fixed a priori. This method has proved popular as it is easily scalable and
+its implementations are concise~\cite{Olafsson2008,Wu2009}. In addition to
+\(k\)-means, hierarchical clustering methods can be effective if a suitable
+number of parts cannot be found initially~\cite{Vuik2016a}. Although, supervised
+hierarchical segmentation methods such as classification and regression trees
+(as in~\cite{Harper2006}) have been used where an existing, well-defined label
+is of particular significance.
+
+\subsubsection{Queuing models}
+
+These are the principal queuing theory works by
+Erlang~\cite{Erlang1917,Erlang1920}. Deadlock is an aspect of applied queuing
+theory of interest in recent literature~\cite{Palmer2018}. The software used is
```

Comment:
If there's a valid reason to reference `Palmer2018` then of course we include it but I don't immediately see how deadlock is related to this work. (As I'm an author on both this and that paper I wouldn't want to just self reference for the sake of it.)

Reply:
Yeah, I've just got it as a placeholder for now 👍